README.md

```diff
@@ -1,2 +1,237 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="./js/images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h2><b>Awesome Books</b></h2>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+
+**[Awesome Books]** "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list.
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML, CSS & Javascript
+- GitHub
+- Node
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[ES6 syntax]**
+- **[Modular JavaScript]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://gealsanchez.github.io/AwesomeModules/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:gealsanchez/AwesomeModules.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-folder
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  cd my-folder
+  npm start
+```
```

Comment:
- [ ] Your project doesn't have any "start" script. I suggest you work on this step. A good example could be to "Open the `index.html` file in a browser". 🤓 🔧